installing/installing_with_agent/installing-with-agent.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="installing-with-agent"]
+= Installing with Agent
+include::_attributes/common-attributes.adoc[]
+:context: installing-with-agent
+
+toc::[]
+
+== Agent installation
+Agent installation is based on the technology of Assisted Installer (AI), that is link:https://console.redhat.com[console.redhat.com] and the Infrastructure Operator. It is a command line tool that generates an image that runs the Assisted Service and the Assisted Installer Agent.
+//TODO>>include telemetry GA
+
+It is a subcommand that resides in the {product-title} installer and also utilizes the {product-title} installer for the input asset management.
+
+This subcommand generates a live {op-system-first} ISO with an embedded Ignition config containing `systemd` units to start the Assisted service, Agent, network, etc. that are running in containers pulled from the {product-title} release payload.
+The Agent installation utilizes Zero Touch Provisioning (ZTP) custom resources. ZTP allows you to provision new edge sites with declarative configurations of bare-metal equipment at remote sites.
```

Comment:
We have also added a mode where the installer takes in higher level manifests, install-config.yaml and agent-config.yaml, to generate the ZTP resources. I think we prefer to have users start with the higher level manifests.  ref >> https://issues.redhat.com/browse/AGENT-248
TODO

Reply:
Ok noted since these are the new implementation details.